实验2部分/shiyan2.py

```
@@ -0,0 +1,462 @@
+            try:
+                filename = os.path.basename(file_path)
+                label = filename.split('_')[0]
+
+                data = np.load(file_path)
+                mfcc_features = data['mfcc']
+
+                self.templates[label].append(mfcc_features)
+                if label not in self.labels:
+                    self.labels.append(label)
+
+            except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)